packages/react-liveness/src/components/FaceLivenessDetector/LivenessCheck/LivenessCameraModule.tsx

```diff
@@ -290,14 +286,32 @@ export const LivenessCameraModule = (
     );
   }
 
+  const shouldShowCenteredLoader = isInitCamera || isInitWebsocket;
```

Comment:
is there a "non-centered" loader?

Reply:
Yep, there is an inline loader that runs when the camera is loading only